.travis.yml

```diff
@@ -1,13 +1,7 @@
-language: node_js
-
-node_js:
-  - '8'
-
 services:
   - docker
-  - rabbitmq
 
-script: npm run test:ci
+script: npm run test
```

Comment:
better yet, let's just use `docker-compose run test` here. That will help us avoid the initial node modules install that travis automatically performs.